start-from-zero/Javascript/src/views/index/index.html

```diff
@@ -2,18 +2,24 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="https://necolas.github.io/normalize.css/7.0.0/normalize.css">
     <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
-    <link rel="stylesheet" href="../sass-architecture/css/main.css">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="../../../../sass-architecture/css/main.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
+    <script src='../common/common.js'></script>
+    <script src='../evaluation/evaluation.js'></script>
+    <script src='../new-evaluation/new-evaluation-a.js'></script>
+    <script src='../login/login.js'></script>
+    
+    
+    
 </head>
 <body>
     <div id="app">
         
     </div>
 </body>
-<script src='login.js'></script>
+<script src='index.js'></script>
```

Comment:
Am a little curious about why you would import this file and the footer when all other js files are imported in the header.